README.md

```diff
@@ -27,9 +27,15 @@ sudo apt update && sudo apt install libpq-dev
 
 ### Run tests
 
-Run tests with `make test`
 App cannot run while testing
 
+Run tests with `make test`
+
+Run api tests with `make test_api`
+
+Before running playwright test for the first time run `yarn install` and `yarn playwright install`
+Run playwright tests with `make test_e2e`
+
 Run simulator tests with `make test_sim_api`
 
```

Comment:
I suggest renaming `make test` -> `make test_unit` and then having `make test` run all three test suites one after the other. What's your opinion on this?

Reply:
good idea! i'll get on that in another branch - expect a pr soon ;)

Reply:
I've opened #60 for this PR :)